### DISCUSSION TITLE
any vanilla mods?(I see the no forge.jar message) + java compatibility

### DISCUSSION THREAD

**Discussion by solanuscasey** [eaglerforge/EaglerForge#52] [Q&A] [answered]
Is eaglerforge compatible with any vanilla mods? and on a side note, this project is directed to from another project that said the client was compatible with java servers. is this true?

**Comment by OeildeLynx31** [comment, marked as answer]
> Is eaglerforge compatible with any vanilla mods? and on a side note, this project is directed to from another project that said the client was compatible with java servers. is this true?

1. Absolutely not
2. Yes it is true, but the current Eaglerforge client isn't compatible, only the old.
